pkg/api/types.go

```diff
@@ -401,6 +401,10 @@ const (
 	ClaimPending PersistentVolumeClaimPhase = "Pending"
 	// used for PersistentVolumeClaims that are bound
 	ClaimBound PersistentVolumeClaimPhase = "Bound"
+	// used for PersistentVolumeClaims that lost their underlying
+	// PersistentVolume. The claim was bound to a PersistentVolume and this
+	// volume does not exist any longer and all data on it was lost.
+	ClaimLost PersistentVolumeClaimPhase = "Lost"
```

Comment:
I think this is fine, as long as it's clear that it's a terminal state where data is lost.  We could also call this `PersistentVolumeClaimEmptyHusk`, since that's basically what the volume is.

To be clear, the only way out of this is 'delete the volume', right?